src/nl/rubensten/texifyidea/structure/latex/LatexStructureViewElement.kt

```diff
@@ -209,6 +206,16 @@ class LatexStructureViewElement(private val element: PsiElement) : StructureView
         }
     }
 
+    private fun addFromLabelingCommands(treeElements: MutableList<TreeElement>, commands: List<LatexCommands>) {
+        val labelingCommands = TexifySettings.getInstance().labelCommands
+        commands.filter { labelingCommands.containsKey(it.commandToken.text) }
+                .forEach {
+                    val element = LatexStructureViewCommandElement.newCommand(it)
+                            ?: return@forEach
```

Comment:
can be oneliner

Reply:
fixed in commit 4fc09f4f03cfda54b13164fb3b1d6e31c694f878